soh/soh/OTRGlobals.cpp

```diff
@@ -549,7 +549,7 @@ extern "C" uint32_t ResourceMgr_GetGameVersion()
     return OTRGlobals::Instance->context->GetResourceManager()->GetGameVersion();
 }
 
-extern "C" uint32_t ResourceMgr_IsGameMasterQuest() {
+uint32_t IsGameMasterQuest() {
```

Comment:
This is a new function technically, but due to where I put it and it just being an extraction of the logic previously in `ResourceMgr_IsGameMasterQuest`, it doesn't appear as a diff other than the signature changing.